setup.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env pwsh
```

Comment:
Does this actually work on a mac? Since I use SQL Local DB that is

Reply:
Narrator: it does not

```
RoundhousE encountered an error.
System.PlatformNotSupportedException: LocalDB is not supported on this platform.
```

Reply:
Yeah, no, it doesn't. But if you use PostgreSQL ... *hack* *cough* *sputter*